addOns/formhandler/src/main/java/org/zaproxy/zap/extension/formhandler/PopupDialogAddField.java

```diff
@@ -78,7 +79,7 @@ protected boolean validateFields() {
     @Override
     protected void performAction() {
         extFormHandler.addFormHandlerFieldName(
-                getNameTextField().getText().toLowerCase(),
+                getNameTextField().getText().toLowerCase(Locale.ROOT),
```

Comment:
I think this can also be removed.

Reply:
Actually, only after tweaking `FormHandlerParam#addField` to lower case when adding to `enabledFieldsNames` (or add the name from the field created earlier).

Reply:
I think I got both these.